README.md

```diff
@@ -69,27 +66,31 @@ namespace netCLIConsoleApp
 ## Using dependency injection:
 
 ```cs
-using inausoft.netCLI;
-using Microsoft.Extensions.DependencyInjection;
-using System;
+...
```

Comment:
Is this required? I hope that when I copy the code it just works.

Reply:
`...` indicates that there are some code missing - and that the example is only partial.

Reply:
How do I know what's missing?